ui/jobs/buff_tracker.ts

```diff
@@ -294,7 +307,18 @@ export class BuffTracker {
         borderColor: '#F28F7B',
         sortKey: 1,
         cooldown: 60,
-        sharesCooldownWith: ['offguard'],
+        sharesCooldownWith: ['offguard', 'peculiarmiss'],
+      },
+      peculiarmiss: {
+        gainAbility: kAbility.PeculiarLight,
+        // durationSeconds: 0 can not trigger the cooldown timer, so use 0.001
+        // you may see an icon flashes by if you gaze at bufftracker.
+        durationSeconds: 0.001,
```

Comment:
This is a neat idea.  I have a couple of small concerns that maybe can be fixed up.

* I don't like the duplication of `peculiar` and `peculiarmiss` in terms of `borderColor` / `sortKey` / `cooldown` / etc etc.
* I wonder if there's a better way to handle `durationSeconds: 0.001`

Mostly I'm wondering if there's some way to combine `peculiar` and `peculiarmiss` and maybe specially mark the ability in a way that doesn't need `durationSeconds: 0.001`.

Can I propose something like making a new `cooldownAbility` field? So, you'd add `cooldownAbility: kAbility.PeculiarLight` to the `peculiar: {` object above.  That field will never be an active use (so no need for `durationSeconds: 0.001`) and will only start the cooldown.  Maybe something else would have to be fixed up to make this work, but that's one possible solution that I can picture.